package.json

```diff
@@ -29,37 +29,43 @@
     "build:ecs": "TARGET=ecs ember build --environment=production"
   },
   "dependencies": {
-    "@esri/arcgis-rest-auth": "^1.7.1",
-    "@esri/arcgis-rest-common-types": "^1.7.1",
-    "@esri/arcgis-rest-request": "^1.7.1",
-    "broccoli-funnel": "^2.0.1",
-    "broccoli-merge-trees": "^2.0.0",
+    "ember-auto-import": "^1.3.0",
     "ember-cli-babel": "^6.16.0",
     "ember-fetch": "^6.2.0",
     "torii": "0.10.1"
   },
+  "peerDependencies": {
+    "@esri/arcgis-rest-auth": "^2.0.0",
+    "@esri/arcgis-rest-portal": "^2.0.0",
+    "@esri/arcgis-rest-request": "^2.0.0",
+    "@esri/arcgis-rest-types": "^2.0.0"
+  },
   "devDependencies": {
     "@ember/optional-features": "^0.6.3",
+    "@esri/arcgis-rest-auth": "^2.0.0",
+    "@esri/arcgis-rest-portal": "^2.0.0",
+    "@esri/arcgis-rest-request": "^2.0.0",
+    "@esri/arcgis-rest-types": "^2.0.0",
     "broccoli-asset-rev": "^2.7.0",
-    "ember-cli": "~3.5.0",
+    "ember-cli": "^3.9.0",
```

Comment:
do we need to bump ember-cli and ember-source to 3.9 at this time? If so, did you use [ember-cli-update](https://www.npmjs.com/package/ember-cli-update) to do it?

Reply:
> do we need to bump ember-cli and ember-source to 3.9 at this time?

nope, when i first opened this PR i was just trying to resolve a few security vulnerabilities.

> did you use ember-cli-update to do it?

also no.

Reply:
hm.. I think we should roll that back then and do it a separate PR. We should _always_ use ember-cli-update to upate Ember.

Reply:
works for me.

i rebased and yanked that commit. this is ready for _re_-review.